db/seeds.rb

```diff
@@ -57,3 +57,10 @@
                     image: "http://cdn.akc.org/content/article-body-image/basset_hound_dog_pictures_.jpg",
                     description: "Hilariously good dog.",
                     shelter: shelter_2)
+review_1 = shelter_1.reviews.create(title: "Found my new best friend!",
+                                    rating: 5,
+                                    content: "Staff were so helpful and the process was easy.")
+review_1 = shelter_1.reviews.create(title: "Good experience",
```

Comment:
Looks like we have two "review_1"s in the seed file.

Reply:
Made this change.